README.adoc

```diff
@@ -78,6 +78,7 @@ Use with `template = 'componentTemplate'`.
 
 Use with `template = 'componentTemplate'`.
 | identifier | boolean | &#10004;&#65039; |  | Defines if the cell should render as header for a11y reasons.
+| columnClass | boolean | &#10004;&#65039; |  | Defines custom `CSS` class for the column it self.
```

Comment:
@welsen: Could you add another option so that we can provide a class for the cells in the column?!